docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.7'
```

Comment:
What's the rationale behind including a docker-compose file here now that we've moved to have a single source of truth by combining the admin service and backend databases?

Reply:
I guess to fully de-couple the two things and also letting them develop independently. 

For example: If the nav-backend breaks the ability to spin up a local db - then admin development can still continue. Arguably - we could move this into the `db-client` in some way in the future. But for now I think this is the simplest/less risky approach.

Happy to discuss further.